client/app/scripts/superdesk-archive/module.js

```diff
@@ -210,6 +210,25 @@ define([
                     additionalCondition:['desks', 'item', function(desks, item) {
                         return desks.getCurrentDeskId() === null;
                     }]
+                })
+                .activity('New Take', {
+                    label: gettext('New Take'),
+                    icon: 'copy',
+                    filters: [{action: 'list', type: 'archive'}],
+                    privileges: {archive: 1},
+                    condition: function(item) {
+                        return (item.lock_user === null || angular.isUndefined(item.lock_user)) && item.state !== 'killed';
+                    },
+                    controller: ['data', '$rootScope', 'desks', 'authoring', 'notify', function(data, $rootScope, desks, authoring, notify) {
+                        authoring.linkItem(data.item, null, null || desks.getCurrentDeskId())
```

Comment:
The second `null` is redundant, the expression always evaluates to `desks.getCurrentDeskId()`.